ui/lib/apps/SearchLogs/components/SearchHeader.tsx

```diff
@@ -119,18 +119,14 @@ export default function SearchHeader({ taskGroupID }: Props) {
         },
       }
 
-      try {
-        const result = await client.getInstance().logsTaskgroupPut(req)
-        const id = result.data.task_group?.id
-        if (!id) {
-          throw new Error('Invalid server response')
-        }
-        navigate(`/search_logs/detail/${id}`)
-      } catch (e) {
-        // FIXME
+      const result = await client.getInstance().logsTaskgroupPut(req)
```

Comment:
What about errors?

Reply:
I think result is OK :smile:

Reply:
What if there are network errors or server side errors?

Reply:
Did you mean `const result = await client.getInstance().logsTaskgroupPut(req)` will throw Error?

I will check it tonight.

Reply:
We have a global axios error handler to handle network and auth error (but only for these 2 kinds of errors). Ideally, we should use try...catch to wrap all API requests, but we don't do that for many API requests. (maybe we should add a wrapper first). I will continue to refine handle error in PR #602

Reply:
Fixed, PTAL~